### ISSUE TITLE
Provide release binary

### ISSUE THREAD

**Issue by DylanRJohnston** [pcwalton/gltf-ibl-sampler-egui#4]
As mentioned on Discord, I have access to a windows machine but it's not set up as a dev environment and I'm very unfamiliar with windows for development. If you'd be able to provide a release binary that would be much appreciated!

**Comment by DylanRJohnston** [comment]
Alternatively if you have instructions on how to setup vulkan properly on mac.

**Comment by jkukatzki** [comment]
yeah I don't get why they're making it so hard for people... "artist-friendly" my ass. I got it to build on mac but am unable to change the vulkan implementation to use moltenVK. I get failed to initialize Vulkan when pressing "generate".
